### PR TITLE
Add ListMergeRequestPipelines

### PR DESCRIPTION
**Pull request by muc**
List all pipelines for a merge request (https://docs.gitlab.com/ce/api/merge_requests.html#list-mr-pipelines)

### CODE REVIEW

File: merge_requests.go [hunk 1]
```diff
@@ -347,6 +347,28 @@ func (s *MergeRequestsService) GetMergeRequestChanges(pid interface{}, mergeRequ
...
+// ListMergeRequestPipelines gets all pipelines for the provided merge request.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/merge_requests.html#list-mr-pipelines
+func (s *MergeRequestsService) ListMergeRequestPipelines(pid interface{}, mergeRequest int, options ...OptionFunc) (PipelineList, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+
```
Comment (svanharmelen) [inline merge_requests.go]: For consistency, could you please remove this blank line?

File: merge_requests.go [hunk 2]
```diff
@@ -347,6 +347,28 @@ func (s *MergeRequestsService) GetMergeRequestChanges(pid interface{}, mergeRequ
...
+	u := fmt.Sprintf("projects/%s/merge_requests/%v/pipelines", url.QueryEscape(project), mergeRequest)
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var p PipelineList
+	resp, err := s.client.Do(req, &p)
```
Comment (svanharmelen) [inline merge_requests.go]: Again for consistency, please add an error check here. I know this is also a valid way to do the same thing, but I like to keep things consistent...

**Comment by muc** [comment]
Sure. I commited those 2 fixes.

**Comment by svanharmelen** [comment]
Thanks @muc! LGTM!

**Pull request merged**